.golangci.yml

```diff
@@ -1,4 +1,102 @@
 ---
 run:
+  build-tags:
+    - btrfs_noversion
+    - libdm_no_deferred_remove
   concurrency: 6
-  deadline: 5m
+linters:
+  enable-all: true
+  disable:
```

Comment:
Honestly I don’t want to be in the business of making these kinds of decisions. The defaults have worked pretty well so far, and related projects with these kinds of long lists have often ended up with long-unmaintained lists with no clear reasons. Compare https://github.com/containers/storage/issues/1579 .

This _also_ disables one of the currently default linters… while simultaneously carrying configuration for it. Let’s not.

Reply:
I changed the config to disable all linters and enable default linters and `whitespace`. Is it ok?

Reply:
7 is better than ~80, but it’s still a list that is going to get out of date.

---

WRT `whitespace` … I aesthetically like removing the empty lines, but the case is not _that_ compelling. The result of adding that linter is that first-time external contributors use their preferred tooling, relying on standard `gofmt` or something like that… and the PR is later rejected, adding another roundtrip and extra friction for that contribution, for a trivial aesthetic reason.